navigation/NavContainer.tsx

```diff
@@ -1,22 +1,27 @@
 import React from 'react';
 
 import { NavigationContainer } from '@react-navigation/native';
+import { useSelector } from 'react-redux';
 
+import { RootState } from '../redux/rootReducer';
 import StartupScreen from '../screens/StartupScreen';
 
 import AppNavigator from './AppNavigator';
 import AuthNavigator from './AuthNavigator';
 
 const NavContainer = () => {
-  // temporary mocked consts
-  const isAuth = true;
+  // temporary mocked const
   const didTryAutoLogin = true;
 
+  const isAuthenticated: boolean = useSelector(
+    (state: RootState) => !!state.auth.token
+  );
+
```

Comment:
Are we going to use useSelector instead reselect?

Reply:
I thought of it. But I decided it will make this PR too messy. We might add it in next PR (along with Immer)